src/main/java/blackjack/domain/GameResult.java

```diff
@@ -0,0 +1,49 @@
+package blackjack.domain;
+
+public enum GameResult {
+    WIN("승"),
+    LOSE("패"),
+    DRAW("무");
+
+    private final String status;
+
+    GameResult(String status) {
+        this.status = status;
+    }
+
+    public static GameResult checkDealerWin(final Player player, final Dealer dealer) {
+        int playerScore = player.calculateTotalCardScore();
+        int dealerScore = dealer.calculateTotalCardScore();
+
+        if (player.isBust()) {
+            return GameResult.WIN;
+        }
+        if (dealer.isBust()) {
+            return GameResult.LOSE;
+        }
+
+        if (dealerScore > playerScore) {
+            return GameResult.WIN;
+        }
+        if (dealerScore < playerScore) {
+            return GameResult.LOSE;
+        }
+        return GameResult.DRAW;
```

Comment:
조건을 간결하게 정리해보면 좋겠어요!
중복된 return 이 보여서요 😃 

```suggestion
        if (player.isBust() || dealerScore < playerScore) {
            return GameResult.WIN;
        }
        ... 생략
        return GameResult.DRAW;
```

Reply:
아 네, 해당 부분은 블랙잭 규칙 우선순위를 구현하는 과정에서 중복된 부분이 발생하게 된 것 같아요. 스텝 2에서는 메서드 구조를 개선해보겠습니다!